utils/UserDTO.ts

```diff
@@ -20,6 +20,12 @@ class UserDTO {
   getYearOfBirth(): number {
     return this.yearOfBirth;
   }
+
+  getGenderNumber(): number {
```

Comment:
put genders and its indexes into a map, and retrieve the data fro the map
example:
```
const genderMap = new Map<string, number>([
    ["Male", 0],
    ["Female", 1]
]);

let index = genderMap.get("Male");  
```